### ISSUE TITLE
Forcehybrid npm - hybrid_lwc - Cordova - Application error: the connection to the server was unsuccessfull

### ISSUE THREAD

**Issue by muheeb75** [forcedotcom/SalesforceMobileSDK-Package#273]
I have tried to build a 'hybrid_lwc' custom salesforce mobile application using "forcehybrid create" package by refering below link: https://developer.salesforce.com/docs/atlas.en-us.mobile_sdk.meta/mobile_sdk/hybrid_lightning_web_components.htm?search_text=lwc

I am not able to run that app in the android emulator as well as in real device, as I am getting "Application Error" as shown in the attachment below.
<img width="183" alt="Android_img" src="https://user-images.githubusercontent.com/65255164/220714464-1bdc036f-d897-4409-84d4-d00eac9ba6a9.PNG">

Please respond to this ASAP.

Thanks.

**Comment by wmathurin** [comment]
Can you successfully connect from a browser to the server that is indicated in the error?

**Comment by arshadaxolt** [comment]
Hi, 
I am also facing the same issue.
do we have any workaround for this?

**Comment by muheeb75** [comment]
When I tried to connect from a browser to the server It was showing me as "Page hellowWorld does not exist".
Please take a look at the screen shot below.
<img width="960" alt="App_Issue1" src="https://user-images.githubusercontent.com/65255164/220883372-d7fa20ae-d813-45f8-bae1-15fcf40952a4.png">

After deploying hellowWorld page, class and LWC components to the server and pushing to the scratch org, It shows empty page in the browser.

Please take a look at the screen shot below.
<img width="958" alt="App_Issue2" src="https://user-images.githubusercontent.com/65255164/220884196-db6be3b9-1b8d-49d6-923f-c57e976f00dd.png">

for mobile still the same issue "Application Error".

Please help me out with this.

Thanks.

**Comment by wmathurin** [comment]
Does it work with the sample org we bundle in the hybrid LWC template: https://github.com/forcedotcom/SalesforceMobileSDK-Templates/tree/dev/HybridLwcTemplate/server ?

**Comment by muheeb75** [comment]
I am currently using the latest forcehybrid package '10.2.0' to create hybrid_lwc. Can you please try with the same package version?

Please refer this link to create latest version forcehybrid lwc app: https://developer.salesforce.com/docs/atlas.en-us.mobile_sdk.meta/mobile_sdk/hybrid_lightning_web_components.htm?search_text=lwc

Can you please reply asap.

Thanks.

**Comment by muheeb75** [comment]
Hi @wmathurin

Can you please help me on this issue, I am still stuck in this same issue.

Can you please reply asap.

Thanks.

**Comment by muheeb75** [comment]
Hi @wmathurin

Can you please help me on this issue.

Can you please reply ASAP.

Thanks.

**Comment by muheeb75** [comment]
Hi @wmathurin

Can you please try with the same package version and share the hybrid_lwc project?

Please refer this link to create latest version forcehybrid lwc app: https://developer.salesforce.com/docs/atlas.en-us.mobile_sdk.meta/mobile_sdk/hybrid_lightning_web_components.htm?search_text=lwc

Can you please reply ASAP.

Thanks.

**Comment by wmathurin** [comment]
The page in the browser is not empty (do a "view page source"). 
It won't show anything in the browser because it needs to go through the container to get its data. See https://github.com/forcedotcom/SalesforceMobileSDK-Templates/blob/dev/HybridLwcTemplate/server/force-app/main/default/lwc/contactList/contactList.js

We just recently released Mobile SDK 11.0 and we tested all our apps and templates including the HybridLwcTemplate successfully on both iOS and Android.
